Replaced old code by fildsets in admin

ar4s opened the pull request:
Old code in licence module is no more useful in new Ralph - is replaced by
inlines in licence's admin.

Connected to #1611

mkurek commented on src/ralph/licences/admin.py:
```diff
@@ -22,6 +41,7 @@ class LicenceAdmin(PermissionAdminMixin, RalphAdmin):
     list_select_related = ['licence_type', 'software_category']
     raw_id_fields = ['software_category', 'manufacturer']
     resource_class = resources.LicenceResource
+    inlines = [BaseObjectLicenceInline, LicenceUserInline]
```
add these inlines to tabs

Commit:
replaced old code by fildsets in admin

Old code in licence module is no more useful in new Ralph - is replaced by inlines in licence's admin.

ghost commented:
:ok:

szok commented:
:ok:
